pkg/sql/export.go

```diff
@@ -135,9 +141,13 @@ func (ef *execFactory) ConstructExport(
 		}
 	}
 
+	exportID := ef.planner.stmt.queryID.String()
+	namePattern := fmt.Sprintf("export%s-%s", exportID, exportFilePatternDefault)
+
 	return &exportNode{
 		source:          input.(planNode),
-		fileName:        string(*fileNameStr),
+		destination:     string(*fileNameStr),
```

Comment:
nit: while you're here, I'd also rename the local vars `fileNameDatum` and `fileNameStr` above to use this more accurate term.

Reply:
@dt Yep. How about the incoming param `fileName` on `ConstructExport`? It’s an interface used in other places. Would it be too far-reaching to rename that as well?

Reply:
Eh, either way. Changing the locals to match what we'd done to the field sent to the physical planner seemed to make sense, but we don't have any changes in this diff to the interface with the optimizer so I'd be fine leaving it as-is there. But don't feel strongly either way.